README.md

```diff
@@ -52,7 +52,10 @@ giving us both centralized and local logs.  (Redundancy is nice).
 		flag.Parse()
 
 		if graylogAddr != "" {
-			gelfWriter, err := gelf.NewWriter(graylogAddr)
```

Comment:
Maybe change function name to `Dial` as `net` package https://golang.org/pkg/net/#Dial with functional options.